pyproject.toml

```diff
@@ -0,0 +1,69 @@
+[project]
+name = 'ras2fim'
+
+    
+maintainers = [
+    {name = "Fernando Salas", email = "fernando.salas@noaa.gov"},
+    {name = "Carson Pruitt", email = "carson.pruitt@noaa.gov"},
+    {name = "Ali Forghani", email = "ali.forghani@noaa.gov"},
+    {name = "Bradford Bates", email = "bradford.bates@noaa.gov"},
+    {name = "Emily Deardorff", email = "emily.deardorff@noaa.gov"},
+    {name = "Fernando Aristizabal", email = "fernando.aristizabal@noaa.gov"},
+    {name = "Greg Petrochenkov", email = "greg.petrochenkov@noaa.gov"},
+    {name = "Hamideh Safa", email = "hamideh.safa@noaa.gov"},
+    {name = "James Coll", email = "james.coll@noaa.gov"},
+    {name = "Matt Luck", email = "matt.luck@noaa.gov"},
+    {name = "Laura Keys", email = "laura.keys@noaa.gov"},
+    {name = "Nick Chadwick", email = "nick.chadwick@noaa.gov"},
+    {name = "Riley McDermott", email = "riley.mcdermott@noaa.gov"},
+    {name = "Rob Gonzalez-Pita", email = "rob.g.pita@noaa.gov"},
+    {name = "Robert Hanna", email = "robert.hanna@noaa.gov"},
+    {name = "Ryan Spies", email = "ryan.spies@noaa.gov"}
+]
+
+requires-python = ">=3.8"
+keywords = ["ras2fim", "mapping", "noaa"]
+license = {path = "./LICENSE"}
+version = "4.4.0"
+
+[project.urls]
+Source = "https://github.com/NOAA-OWP/ras2fim"
+Wiki = "https://github.com/NOAA-OWP/ras2fim/wiki"
+
+
+# - Tools -
+
+[tool.black]
+line_length = 110
+targer-version = ['py38']
```

Comment:
`target-version = ['py38']`?

Reply:
removed it as it has been tested and I assume it was ignored. I added it during some tests to upgrade python 3.8 but rolled it back.